### ISSUE TITLE
%help magic does not work for Stata commands

### ISSUE THREAD

**Issue by arnonerba** [kylebarron/stata_kernel#426]
The `%help` magic does not currently seem to work for Stata commands. Built-in commands like `%help magics` work, but Stata-specific queries like `%help histogram` do not. The cell is marked as `[*]` in JupyterLab and never returns any output.

I can confirm this behavior on an Ubuntu 20.04 system running Stata 17 and on a Windows system running Stata 15. The stata_kernel debug log is empty.

**Comment by mcaceresb** [comment]
Just to confirm, do you have internet?

**Comment by arnonerba** [comment]
Yep, I have Internet access, and `help` commands work properly in the Stata app itself (at least on the Ubuntu 20.04 machine).

**Comment by poisoners** [comment]
Also need to add `import requests` at the beginning.

I'm not sure why, but using `requests` to fetch the webpage works fine, while using `urllib` does not.
